styles/app.scss

```diff
@@ -1,6 +1,20 @@
 // @import 'yoga';
 // @import 'normalize';
 
-// $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
-// @import 'font-awesome';
+$fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
+
+@import 'font-awesome';
 @import 'box-sizing';
+@import 'variables';
+@import 'top-nav';
+@import 'cssgram';
+@import 'main';
+@import 'footer';
+@import 'city';
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  max-width: 100vw;
+  min-height: 100vw;
```

Comment:
Max width and min height are only needed if you are going to do a sticky footer using flex-direction column.